fusor-ember-cli/app/routes/openstack/overcloud.js

```diff
@@ -28,25 +28,30 @@ export default Ember.Route.extend({
   },
 
   deactivate() {
-    this.updateNeutronPublicInterface();
-    this.send('updatePlanParameters', [{
-      name: 'Controller-1::NeutronPublicInterface',
-      value: this.get('controller').get('neutronPublicInterface')
-    }]);
+    var controller = this.get('controller'),
+      changedParams = [
+        {
+          name: 'Controller-1::NeutronPublicInterface',
+          value: this.get('controller').get('neutronPublicInterface')
+        },
+        {
+          name: 'Controller-1::AdminPassword',
+          value: this.get('controller').get('overcloudPassword')
+        }
+      ];
+
+    this.updateLocalPlanParameters(changedParams);
+    this.send('updatePlanParameters', changedParams);
     return this.send('saveDeployment', null);
   },
```

Comment:
@cfchase, yes the `deactivate` will be trigger and save invalid info, but when the user comes back, he will see info is invalid. Afaik, the important thing is that all inputs are valid before a user clicks "Deploy"